specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/OperationalInsights.json

```diff
@@ -815,11 +872,126 @@
             }
           }
         }
+      },
+      "patch": {
+        "tags": [
+          "Workspaces"
+        ],
+        "x-ms-examples": {
+          "WorkspacesPatch": { "$ref": "./examples/WorkspacesUpdate.json" }
+          },        
+        "operationId": "Workspaces_Update",
+        "description": "Updates a workspace.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the workspace."
+          },
+          {
+            "name": "workspaceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[A-Za-z0-9][A-Za-z0-9-]+[A-Za-z0-9]$",
+            "minLength": 4,
+            "maxLength": 63,
+            "description": "The name of the workspace."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Workspace"
+            },
+            "description": "The parameters required to patch a workspace."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The existing workspace was successfully updated. Check provisioningStatus to see detailed status.",
+            "schema": {
+              "$ref": "#/definitions/Workspace"
+            }
+          }
+        }
+      }      
+    },
+    "/providers/Microsoft.OperationalInsights/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "x-ms-examples": {
+          "OperationsList": { "$ref": "./examples/OperationsList.json" }
+          },        
+        "operationId": "Operations_List",
```

Comment:
Seeing as this returns a list, it should be marked as `x-ms-pageable`. I can help hook you up with examples of how to do that if necessary.

Reply:
Added x-ms-pageable. Please see next commit